bin/download

```diff
@@ -34,7 +34,7 @@ download_golang() {
 
   platform=$(get_platform)
   arch=$(get_arch)
-  download_url="https://dl.google.com/go/go${version}.${platform}-${arch}.tar.gz"
+  download_url="https://storage.googleapis.com/golang/go${version}.${platform}-${arch}.tar.gz"
```

Comment:
```suggestion
  download_url="https://go.dev/dl/go${version}.${platform}-${arch}.tar.gz"
```

Reply:
I tried that originally but that site didn't seem to have the sha256 available (in the expected file name format)

Reply:
Ah, I see.